packages/plugin-core/src/_extension.ts

```diff
@@ -188,6 +385,41 @@ class ExtensionUtils {
       });
     }
   }
+
+  // run this when we don't have an active workspace
+  static watchForNativeWorkspace(ext: DendronExtension) {
```

Comment:
We could just drop this if you'd prefer, it has been deprecated and is not needed anymore. Feel free to keep it in this PR if that's easier though.

Reply:
updated